tests/tests.yaml

```diff
@@ -1,6 +1,6 @@
 answer_tests:
 
-  local_art_002: # PR 3081
+  local_art_003: # PR 3081, 3101
```

Comment:
Required as the old answer tests had duplicate entries (with a different format). This was causing the tests to fail as there are no duplicate entries any more.

Reply:
fascinating